### DISCUSSION TITLE
✅  Cmd + Tab key

### DISCUSSION THREAD

**Discussion by ap2de** [isontheline/pro.webssh.net#501] [Q&A] [answered]
On my ipad i use Cmd + Tab key to switch between apps.
Using websshIcan not swith to another app. Rather I receive the possibiity to search for a server.
What preference have I got to change as to be able to switch to another app when using webssh?
Thank you for your reply.

**Comment by isontheline** [comment, marked as answer]
I just tried `Cmd + Tab` when on the server list and when on the terminal and I can switch to other apps as well.

Strange :/

Have you tried to reboot your device?

**Reply by ap2de** [reply]
after reboot it worked.
i didi not know this was necessary.

**Reply by isontheline** [reply]
Perhaps a bug inside WebSSH (and or iOS). 

Please report this issue again if the trouble comes again.

**Comment by florian1984** [comment]
I‘ve same issue on latent iPad OS 26. cmd + tab Not working.  Habe to use cmd + Space to change Apps. Please fix it.

**Reply by isontheline** [reply]
Does cmd+space is the same as cmd+tab?

**Reply by florian1984** [reply]
Cmd+Space is on IPad  the shortcut to change Apps. Cmd+Space which is working is the command Lauscher to start any program.

**Reply by isontheline** [reply]
Does cmd+tab isn't working only on the terminal view?

**Reply by florian1984** [reply]
Strange behavior. I just checked where exactly it's happening. But I couldn't reproduce the error on my iPad. Now it's working without any problems. I'll get back to you if I can determine the cause. Thanks you.

**Reply by isontheline** [reply]
Thank you a lot @florian1984 for your feedback 🙏
No problem, feel free to come back at any time

Have a great day ☀️
